src/Components/LocationCard.js

```diff
@@ -32,8 +32,9 @@ import star from '../assets/map-pins/star.png';
 import taxi from '../assets/map-pins/taxi.png';
 import wc from '../assets/map-pins/wc.png';
 import trash from '../assets/trash.svg';
-import mapPin from '../lib/icons';
-export default function LocationCard({ location }) {
+/* import mapPin from '../lib/icons'; */
```

Comment:
It looks like you don't need this import, maybe you should consider deleting it? :)